### ISSUE TITLE
Incorrect '"bars" visualization when Zabbix's "Custom Invervals -> Scheduling" is used

### ISSUE THREAD

**Issue by mbrekhov** [grafana/grafana-zabbix#1187]
Hi,

Not sure if it is Grafana bug, or Zabbix-plugin.

### Example 1

Here is an example how Zabbix interval is configured:
![image](https://user-images.githubusercontent.com/41376938/114019354-f8a41d80-9876-11eb-996a-8be9be9e3262.png)

This is correct visualization(24h):
![image](https://user-images.githubusercontent.com/41376938/114019761-7700bf80-9877-11eb-8d18-653b12ce2e5a.png)

But when I change to 2d it is showing up incorrectly:
![image](https://user-images.githubusercontent.com/41376938/114019671-5d5f7800-9877-11eb-999b-a2526dca4ac9.png)

But I've found solution how to fix this on client side. Need to set "Min interval":
![image](https://user-images.githubusercontent.com/41376938/114020126-e676af00-9877-11eb-8ef8-4a3b1f2b424f.png)

And now the same 2d is showing up correctly:
![image](https://user-images.githubusercontent.com/41376938/114172320-3029cd00-993e-11eb-9f40-db1b109aff9d.png)

### Example 2:

**24 hours** in Grafana:
![image](https://user-images.githubusercontent.com/41376938/114023238-4d499780-987b-11eb-8dd7-2e5dcfe26c05.png)

**2days** in Grafana(now-2d):
![image](https://user-images.githubusercontent.com/41376938/114023271-59cdf000-987b-11eb-8310-bd1178c06091.png)

**3 days** in Grafana(now-3d):
![image](https://user-images.githubusercontent.com/41376938/114023323-681c0c00-987b-11eb-89b0-15f0bdd7bd62.png)

Also I'm able to fix this by using sum(5m) function:
![image](https://user-images.githubusercontent.com/41376938/114023607-c2b56800-987b-11eb-91d9-f1c56b575ce5.png)

**Software versions**

| Grafana | Zabbix | Grafana-Zabbix Plugin |
| ------- | ------ | --------------------- |
| 7.5.1   | 5.2.6 | 4.1.4                 |

**Comment by mbrekhov** [comment]
It seems that the issue is not relates to  "Custom Invervals". I observe the same behavior in "usual" intervals too.

Looks like "data alignment" is the root cause of the issue.

**Comment by alexanderzobnin** [comment]
Yes, I think you're right - this is related to data alignment (in some cases it uses interval detection). I'll take a look.

**Comment by alexanderzobnin** [comment]
@mbrekhov could you test this against the latest version?

**Comment by mbrekhov** [comment]
I cannot due to incompatibility of this plugin with our Grafana's version (7.5.x)
